diff --git a/setup.cfg b/setup.cfg
index a6739f8..993380b 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -31,7 +31,7 @@ keywords =
 
 [options]
 python_requires = >=3.6
-packages = ansible_bender
+packages = find:
 include_package_data = True
 
 setup_requires =
@@ -43,6 +43,11 @@ install_requires =
     tabulate
     jsonschema
 
+[options.packages.find]
+exclude =
+    tests
+    tests.*
+
 [options.extras_require]
 testing =
     pytest

